src/iexamine.cpp

```diff
@@ -1516,6 +1516,102 @@ void iexamine::aggie_plant(player *p, map *m, int examx, int examy)
     }
 }
 
+// Highly modified fermenting vat functions
```

Comment:
Not necessary for this PR, but we need to take a look at what code is shared here and pull it into helpers or something, or even better push the differences to json and make it a shared handler.